Bookish/Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -43,23 +43,8 @@
 </div>
 
 
-@if (Model.MaxPages > 1)
+@if (Model.TotalPages > 1)
 {
-    <nav>
-        <ul class="pagination">
-            <li class="page-item @Model.GetDisabledPrevious()">
-                <a class="page-link" asp-action="Catalogue" asp-route-pageNumber="@(Model.PageNumber - 1)" asp-route-searchTerm="@Model.SearchTerm">Previous</a>
-            </li>
-            @for (var i = 1; i <= Model.MaxPages; i++)
-            {
-                <li class="page-item @(Model.GetActiveClass(i))">
-                    <a class="page-link" asp-action="Catalogue" asp-route-pageNumber="@i" asp-route-searchTerm="@Model.SearchTerm">@i</a>
-                </li>
-            }
-            <li class="page-item @Model.GetDisabledNext()">
-                <a class="page-link" asp-action="Catalogue" asp-route-pageNumber="@(Model.PageNumber + 1)" asp-route-searchTerm="@Model.SearchTerm">Next</a>
-            </li>
-        </ul>
-    </nav>
+    <partial name="_Pagination.cshtml" />
```

Comment:
I don't think you need the `.cshtml`? And I would probably to the `if` in the partial as well but up to you, either is fine really 🙂